Runtime/Scripts/DeviceInfoIntegration.cs

```diff
@@ -64,11 +64,10 @@ enum NetworkDataType
       CarrierInfoClass carrierInfo = new CarrierInfoClass();
       Dictionary<string, string> map;
       int sdk_int = carrierInfo.getAndroidSDKVers();
-      if (UnityEngine.XR.XRSettings.isDeviceActive && UnityEngine.XR.XRSettings.loadedDeviceName.Contains("oculus"))
+      if (UnityEngine.XR.XRSettings.enabled)
```

Comment:
Is XR never associated with Android Phones and iOS?

Reply:
Fixed to check for Oculus only